src/resources/views/crud/inc/form_fields_script.blade.php

```diff
@@ -49,11 +49,13 @@ class CrudField {
 
         enable(e) {
             this.input.removeAttr('disabled');
+            this.input.trigger('backpack_field.enabled');
             return this;
         }
 
         disable(e) {
             this.input.attr('disabled', 'disabled');
+            this.input.trigger('backpack_field.disabled');
```

Comment:
Odd to see underscore in a JS event name. I'd rename these events to something like `backpack:field.enabled` - what do you think? Odd naming convention? Followed [this](https://stackoverflow.com/questions/19071579/javascript-dom-event-name-convention).

Reply:
If we add events for these two, we should add events for more/all methods. Does it make sense to make them past tense (action already happened), or should we just go with the imperative?

- `backpack:field.enable`
- `backpack:field.disable`
- `backpack:field.hide`
- `backpack:field.show`
- `backpack:field.require`
- `backpack:field.unrequire`